incubator/mariadb-cluster/templates/NOTES.txt

```diff
@@ -19,12 +21,15 @@ To connect to your database
 
   1. Run a pod that you can use as a client:
 
-      kubectl run {{ template "fullname" . }}-client --rm --tty -i --image {{ .Values.image }} --namespace {{ .Release.Namespace }} --command -- bash
+      kubectl run {{ template "fullname" . }}-client --rm --tty -i --image  {{ template "mariadb.image" . }} --namespace {{ .Release.Namespace }} --command -- bash
 
   2. To connect to master service (read/write):
 
       mysql -h {{ template "master.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local -uroot -p {{ .Values.db.name }}
 
+{{- if .Values.replication.enabled }}
+
   3. To connect to slave service (read-only):
 
       mysql -h {{ template "slave.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local -uroot -p {{ .Values.db.name }}
+{{- end }}
```

Comment:
Currently we added information about how to connect remotely to MariaDB on stable chart and a Waring message. See:

- https://github.com/kubernetes/charts/pull/4869
- https://github.com/kubernetes/charts/blob/master/stable/mariadb/templates/NOTES.txt

Please add it here too.